queries/dev/run.py

```
@@ -1,11 +1,261 @@
+
+# run_dbt_tests(  # ok
+#     dataset_id="dashboard_subsidio_sppo",
+#     _vars={"date_range_start": "2023-10-01 00:00:00", "date_range_end": "2024-01-31 00:00:00"},
+#     flags="--target hmg",
+# )
+
+
+# -*- coding: utf-8 -*-
+# import os
+
+from utils import fetch_dataset_sha, run_dbt_model
```

Ruff (0.11.9)

Redefinition of unused `run_dbt_model` from line 6

(F811)